go.sum

```diff
@@ -68,7 +68,7 @@ github.com/hashicorp/go-azure-helpers v0.0.0-20181211121309-38db96513363 h1:9d0j
 github.com/hashicorp/go-azure-helpers v0.0.0-20181211121309-38db96513363/go.mod h1:Y5ejHZY3jQby82dOASJzyQ2xZw37zs+D5x6AaOC6O5E=
 github.com/hashicorp/go-cleanhttp v0.0.0-20170211013415-3573b8b52aa7 h1:67fHcS+inUoiIqWCKIqeDuq2AlPHNHPiTqp97LdQ+bc=
 github.com/hashicorp/go-cleanhttp v0.0.0-20170211013415-3573b8b52aa7/go.mod h1:JpRdi6/HCYpAwUzNwuwqhbovhLtngrth3wmdIIUrZ80=
-github.com/hashicorp/go-getter v0.0.0-20180226183729-64040d90d4ab h1:ap3umB+Fouf/i1/N1q0AZLjOxSg1s8RIQ159wc7wNPE=
+github.com/hashicorp/go-getter v0.0.0-20180226183729-64040d90d4ab h1:q9prbneWtMiyRZWcIZM5gFEIp7ROi+/iXd1fbLTY9T4=
```

Comment:
I've run `go mod vendor` and can't reproduce this, where are you seeing this issue?

Reply:
I always see this error in my MacBook Pro, even the module cache folder, $GOPATH/pkg/mod, is cleaned.

Reply:
This seems to be a known issue in Go before 1.11.4:
https://github.com/golang/go/issues/29281

Reply:
maybe that is why i am not seeing the issue, running go 1.11.4 here

Reply:
@metacpp please upgrade to Go 1.11.4 to resolve this issue